crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -364,7 +397,7 @@ pub trait DatabaseExt: Database<Error = DatabaseError> {
     fn record_cheatcode_purity(&mut self, cheatcode_name: &'static str, is_pure: bool);
 }
 
-struct _ObjectSafe(dyn DatabaseExt);
+// struct _ObjectSafe(dyn CheatcodeBackend);
```

Comment:
Did you intend to comment this?

Reply:
No, commented back in [343fa19](https://github.com/NomicFoundation/edr/pull/901/commits/343fa19efeaa90f9cbf374522109a1ca1920d626)